examples/htlc-pyteal-ts/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "algosdk": "^1.9.1"
+    "algosdk": "scale-it/js-algorand-sdk.git#f0b9dd9bf45827e0b448a2f6a7dd8c0735317d24"
```

Comment:
The type changes are now released - [https://github.com/algorand/js-algorand-sdk/releases/tag/v1.10.1](https://github.com/algorand/js-algorand-sdk/releases/tag/v1.10.1). Can we use `^1.10.1` here?

Reply:
no, there is one issue in types of js-sdk, I have updated in scaleit. Will update it here.